index.html

```diff
@@ -37,50 +37,44 @@ <h2>Portfolio</h2>
 			<div class="portfolio-card-container">
```

Comment:
looks like your indentation might be wrong here, try removing a couple of levels